lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -871,6 +873,14 @@ def _post(self, data):
         url = reverse('verify_student_create_order')
```

Comment:
That's fine, I remember our conversation now. To preserve our sanity, it might be worth adding a line to the docstring for this test class explaining that verify_student's `create_order` sometimes uses the old shoppingcart but will instead call out to Otto if the requested course mode is associated with a SKU.